lib/puppet/type/scheduled_task.rb

```diff
@@ -89,6 +89,7 @@ def insync?(current)
       2003 computer."
 
     newvalue(1)
+    newvalue(2)
```

Comment:
Ensure the new compatibility mode is allowed. This will require `/spec/unit/puppet/type/scheduled_task_spec.rb:134` to be updated now that the value is allowed.

Reply:
Done in https://github.com/puppetlabs/puppetlabs-scheduled_task/pull/27/commits/8373a8897bc9e22c2bc43c0086a67bc8eae76efd